cart/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/cart.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
```

Comment:
As Dani went over in class, adding the links to your other pages will allow it to show in Github pages.

Reply:
Yes I tried so hard last night and couldn’t get it I wasn’t much off but was off I actually got that done before lunch.... 😊